### PR TITLE
MythicsoftLtd.FileLocator version 9.1.3389.1

### PR DESCRIPTION
**Pull request by landmn**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/105053)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230503100323

**Comment by stephengillie** [comment]
This application appears to be an MSI within a self-extracting ZIP file. This usually precludes passing installer switches, as the ZIP extractor ignores them. 

image

The [download](https://www.mythicsoft.com/filelocatorpro/download/) page has the full MSI - maybe that should be used instead?

File: manifests/m/MythicsoftLtd/FileLocator/9.1.3389.1/MythicsoftLtd.FileLocator.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: MythicsoftLtd.FileLocator
+PackageVersion: 9.1.3389.1
+Installers:
+- InstallerUrl: https://download.mythicsoft.com/flp/3389/filelocator_3389.exe
+  Architecture: x86
+  InstallerType: nullsoft
+  InstallerSha256: 98B5654ABE82CEB0F56E4C8AF9F7AFE928D44F113DF2DCA83231BB1575817242
```
Comment (stephengillie) [inline manifests/m/MythicsoftLtd/FileLocator/9.1.3389.1/MythicsoftLtd.FileLocator.installer.yaml:10]:
```suggestion
- InstallerUrl: https://download.mythicsoft.com/flp/3389/filelocator_x64_msi_3389.zip
  Architecture: x64
  InstallerType: wix
  InstallerSha256: 1A187A8F618E254FEA67BBFD52023CD43803529FB127702E71E010603527F99C
```

Comment (stephengillie) [reply manifests/m/MythicsoftLtd/FileLocator/9.1.3389.1/MythicsoftLtd.FileLocator.installer.yaml:10]: The `x86` MSI is a 16-bit application that won't install on 64-bit OSes. 
The `x64` MSI isn't a valid package.
image

**Review by ItzLevvie — APPROVED** [review]

**Comment by landmn** [comment]
I have sole ownership of intellectual property rights to my Submissions and
I am not making Submissions in the course of work for my employer.

**Comment by landmn** [comment]
@microsoft-github-policy-service agree

**Pull request closed without merging**